.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -1,5 +1,6 @@
 <component name="ProjectCodeStyleConfiguration">
   <state>
     <option name="USE_PER_PROJECT_SETTINGS" value="true" />
+    <option name="PREFERRED_PROJECT_CODE_STYLE" value="Default" />
```

Comment:
Shouldn't we keep the code style preferred to be per-project rather than the user's default?

Reply:
Actually, this is again that same issue, this file automatically gets created as well updated in android studio, even Nikita is facing this same issue.